server/.env

```
@@ -1,2 +1,2 @@
 PORT=5000
-DBURL=mongodb://localhost:27017/enquiry_db
\ No newline at end of file
+DBURL=mongodb+srv://admin2000:Karan54321@kartik.btjjy4d.mongodb.net/
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)